docs/account_management/cheaha_account.md

```diff
@@ -39,7 +39,7 @@ If your account is in good standing you should see a page like the following.
 
 ### Account Requires Certification
 
-We review accounts periodically to ensure the system is being used fairly and as intended. Part of this process is to certify that researchers with accounts still wish to make use of Cheaha. Once per year every researcher will be required to certify their account before making use of Cheaha. In addition, an account may also require certification after it has been placed on hold (see our [Account on Hold section](#account-on-hold)). This ensures that you, the user, are engaged in the process of reactivating your account to use our resources. If your account requires certification, when logging in to our [Open OnDemand](../cheaha/open_ondemand/index.md) web portal you will see the following notification page.
+We review accounts periodically to ensure the system is being used fairly and as intended. Part of this process is to certify that researchers with accounts still wish to make use of Cheaha. Once per year every researcher will be required to certify their account before making use of Cheaha. In addition, all accounts require certification after being placed on hold (see our [Account on Hold section](#account-on-hold)). This way, we ensure users are engaged in the process of reactivating their account to use our resources. If your account requires certification, you will see the below notification page, when logging into our [Open OnDemand](../cheaha/open_ondemand/index.md) web portal.
```

Comment:
The last sentence "If..., you will see..., when logging...". Remove the omma after "page" as the following clause is part of the previous clause. If you want to separate the clauses, you could alternatively write it in the following order:

> When logging into ..., if your account..., you will see...